### DISCUSSION TITLE
springboot/aop-log

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#139] [Announcements]
# springboot/aop-log

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/springboot/aop-log.html

**Comment by 2firenut** [comment]
没人吗

**Comment by weh-coder** [comment]
## 来了呀
